frontend/src/components/AdminEditModal/AdminEditModal.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Button, Modal, ModalHeader, ModalBody, Form, FormGroup, Label, Col, Input, Alert,
+} from 'reactstrap';
+import './AdminEditModal.css';
+import { adminEditUser } from '../../lib/apiClient';
+
+
+class AdminEditModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      alertVisible: false,
+    };
+
+    this.onDismiss = this.onDismiss.bind(this);
+  }
+
+  onDismiss() {
+    this.setState({ alertVisible: false });
+  }
+
+  // check if values in the form have changed
+  compareUserData = (event) => {
+    event.preventDefault();
+    const { selectedUser, toggleEditModal, refreshUsers } = this.props;
+    const newData = event.target.elements;
+    const oldData = selectedUser;
+
+    if (newData.email.value === oldData.email
+    && newData.firstName.value === oldData.first_name
+    && newData.lastName.value === oldData.last_name) {
+      this.setState({ alertVisible: true });
+    } else {
+      adminEditUser(newData, selectedUser.id).then(() => {
+        toggleEditModal();
+        refreshUsers();
+      });
+    }
+  }
+
+  render() {
+    const {
+      editModal, toggleEditModal, selectedUser,
+    } = this.props;
+    const { alertVisible } = this.state;
+
+    const closeBtn = <button type="button" className="edit-user-close-button" onClick={() => { toggleEditModal(); this.onDismiss(); }}><img src="https://img.icons8.com/windows/32/9b9b9b/cancel.png" alt="close" /></button>;
```

Comment:
Rather than one-lining that whole onClick, let's abstract the onClick out to an independent function and put the `img` tag on its own line between the button tags.